code/modules/admin/topic.dm

```diff
@@ -4786,7 +4786,7 @@
 					return FALSE
 
 				if (!preacher)
-					to_chat(world, "<span class='warning'>No mob selected.</span>")
```

Comment:
oof

Reply:
Yeah, it's been here since months
Good thing nobody used that feature